pkgs/development/python-modules/restinstance/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, fetchFromGitHub, buildPythonPackage, jsonschema, nox }:
+
+buildPythonPackage rec {
+  pname = "RESTinstance";
```

`pname`s should be normalized, see https://nixos.org/manual/nixpkgs/stable/#contributing

```suggestion
  pname = "restinstance";
```